src/Page/TextLayer.css

```diff
@@ -26,33 +26,20 @@
   }
 }
 
-[data-main-rotation='90'] {
-  transform: rotate(90deg) translateY(-100%);
-}
-[data-main-rotation='180'] {
-  transform: rotate(180deg) translate(-100%, -100%);
-}
-[data-main-rotation='270'] {
-  transform: rotate(270deg) translateX(-100%);
-}
-
 .textLayer {
   position: absolute;
   text-align: initial;
-  left: 0;
-  top: 0;
-  right: 0;
-  bottom: 0;
+  inset: 0;
   overflow: hidden;
+  opacity: 0.25;
```

Comment:
This line was intentionally removed - avoids custom selection color which serves no good purpose